_targets.R

```diff
@@ -0,0 +1,40 @@
+library(targets)
+src_files <- Sys.glob('*/src/*.R')
+for (f in src_files) {
+  print(paste("sourcing", f))
+  source(f)
+}
+tar_option_set(packages = c("miceadds", "readr", "stringr",
+                            "tidyverse", "sbtools", "whisker"))
+
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
+    download_data(out_filepath = "1_fetch/out/model_RMSEs.csv"),
```

Comment:
If someone else tries to run this, the pipeline will fail because `1_fetch/out/` doesn't exist in the repository. Committing an empty directory to git requires some trickery, though! One convention, which we often use here, is to create an empty file called `.gitkeep` or `.empty` in the directory in question. Committing this file will add the directory to the repository.

So you could do something like
```bash
touch 1_fetch/out/.gitkeep
git add -f 1_fetch/out/.gitkeep # -f is necessary since you have */out/* in .gitignore
```